src/client/components/molecules/portfolioSteps/Job/index.tsx

```diff
@@ -36,29 +35,11 @@ const JobStep = ({ changeRoute }: JobProps) => {
             <div className="jobForm__title">Work Experencie</div>

```

Comment:
even tho this was added in the past, can you please enforce the BEM pattern here?
`classNames should be with kebab-case instead of camelCase`

suggestin: job-form - job-form__title 


...if we can get rid of the "form" word, better